src/components/tabs/themes/light/tab.base.scss

```
@@ -0,0 +1,74 @@
+@use '../../../../styles/common/component';
+@use '../../../../styles/utilities' as *;
+
+$hover-background: var(--hover-background, color(gray, 200)) !default;
+
+:host {
+    @include type-category('subtitle-2');
+
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    min-width: rem(90px);
+    max-width: rem(360px);
+    word-wrap: break-word;
+    padding: rem(12px) rem(16px);
+    overflow: hidden;
+    cursor: pointer;
+    position: relative;
+    transition: all .3s cubic-bezier(.35, 0, .25, 1);
+    user-select: none;
+    background: color(surface, 500);
+    color: color(gray, 700);
+
+    > * {
+        margin-inline-start: rem(12px);
+
+        &:first-child {
+            margin-inline-start: 0;
+        }
+    }
```

We can replace all this with:
```suggestion
    gap: rem(12px);
```